index.bs

```diff
@@ -6264,6 +6264,36 @@ This attestation statement format is exclusively used by Apple for certain types
     5. Verify that the [=credential public key=] equals the Subject Public Key of |credCert|.
     6. If successful, return implementation-specific values representing attestation type [=Anonymization CA=] and attestation trust path |x5c|.
 
+## Compound Attestation Statement Format ## {#sctn-compound-attestation}
+
+The "compound" attestation statement format is used to pass multiple, self-contained attestation statements in a single ceremony.
+
+: Attestation statement format identifier
+:: compound
+
+: Attestation types supported
+:: [=Basic=], [=AttCA=], [=Anonymization CA=]
+
+: Syntax
+:: The syntax of a compound attestation statement is defined as follows:
+
+    ```
+    $$attStmtType //= (
+                          fmt: "compound",
+                          attStmt: compoundAttStmts
+                      )
+
+    compoundAttStmts = []
```

Comment:
How does this constrain the shape of the potential entries in `compoundAttStmts` to one of the following `$$attStmtType` that are currently in the spec?

```
$$attStmtType //= ( fmt: "packed", attStmt: packedStmtFormat )
$$attStmtType //= ( fmt: "tpm", attStmt: tpmStmtFormat )
$$attStmtType //= ( fmt: "android-key", attStmt: androidStmtFormat )
$$attStmtType //= ( fmt: "android-safetynet", attStmt: safetynetStmtFormat )
$$attStmtType //= ( fmt: "fido-u2f", attStmt: u2fStmtFormat )
$$attStmtType //= ( fmt: "apple", attStmt: appleStmtFormat )
```

The three "Attestation types supported" mentioned don't obviously connect to any of these existing statement types which is why I ask.

Reply:
Good call. I'm wondering if I should just remove the "Attestation types supported" altogether. Thoughts @MasterKale?

Reply:
Yeah, perhaps. Maybe replace the three that are there now with wording that constrains the entries to one of those other existing types?

Reply:
@MasterKale does https://github.com/w3c/webauthn/pull/1950/commits/21ae2d18bfcef93b1d26fcce0e63218a4ad26345 address this?

Reply:
Yeah, looks good to me 👍